src/audio/detect_test.c

```diff
@@ -37,11 +37,15 @@
 #define tracev_keyword(__e, ...) \
 	tracev_event(TRACE_CLASS_KEYWORD, __e, ##__VA_ARGS__)
 
+#define INT24_MAX (0xFFFFFF / 2)
```

Comment:
It's already defined in sof/audio/format.h.

Reply:
Ok, I will use that one them.